pda.sh

```diff
@@ -0,0 +1,42 @@
+package: PDA
+version: "%(tag_basename)s"
+tag: 11.0.7
+source: https://github.com/PascalBoeschoten/pda
```

Comment:
@PascalBoeschoten : here are the instructions to transfer your pda repo to the AliceO2Group organization. Could you try it ? I think that you have the right to create a repo there so it should work.